ts/models/message.ts

```diff
@@ -1182,12 +1213,15 @@ export class MessageModel extends Backbone.Model<MessageAttributes> {
     }
   }
 
-  private findAndFormatContact(pubkey: string): FindAndFormatContactType {
+  public findAndFormatContact(pubkey: string): FindAndFormatContactType {
     const contactModel = getConversationController().get(pubkey);
     let profileName: string | null = null;
     let isMe = false;
 
-    if (pubkey === UserUtils.getOurPubKeyStrFromCache()) {
+    if (
+      pubkey === UserUtils.getOurPubKeyStrFromCache() ||
+      (pubkey && pubkey.startsWith('15') && isUsAnySogsFromCache(pubkey))
```

Comment:
I think the `isUsAnySogsFromCache` already does the `pubkey === UserUtils.getOurPubKeyStrFromCache` test inside